packages/core/src/browser/core-preferences.ts

```diff
@@ -97,6 +97,12 @@ export const corePreferenceSchema: PreferenceSchema = {
             'description': nls.localizeByDefault('Controls whether editors showing a file that was opened during the session should close automatically when getting deleted or renamed by some other process. Disabling this will keep the editor open  on such an event. Note that deleting from within the application will always close the editor and that dirty files will never close to preserve your data.'),
             'default': false
         },
+        'workbench.editor.mouseBackForwardToNavigate': {
+            'type': 'boolean',
+            // nls-todo: Will be available with newer VSCode API
```

Comment:
When it's available, this should be caught automatically by the linter (using `localizeByDefault` is an error if the string isn't available; using just `localize` is an error if the exact string is available).

Reply:
Thank you for the clarification! I noticed that I got the error with `localizeByDefault` and checked what it is you are doing in the `localization-check.js`, it's very nice!